### DISCUSSION TITLE
add FooterRow to Theme

### DISCUSSION THREAD

**Discussion by proddy** [table-library/react-table-library#65] [Ideas]
any plans to extend the `Theme` type to include `FooterRow`? It is a GridView property.

**Comment by rwieruch** [comment]
Not yet. What use cases are there that I need to take care off when integrating a proper footer row?

**Reply by proddy** [reply]
just the styling, I want to add a border

**Reply by rwieruch** [reply]
The new version [4.0.8](https://github.com/table-library/react-table-library/releases/tag/v4.0.8) has new properties on the theme such as `Footer`, `FooterRow`, and `Footer` now. Let me know if everything works as expected :)

**Reply by proddy** [reply]
cool, I'll try it out tonight. thanks!

**Reply by rwieruch** [reply]
And let me know what else you would see for this library :) Or any other feedback to improve it! Thanks @proddy 🚀
